### PR TITLE
fix(test): adding Che workspace test

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

File: booster_bdd/che.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# -DosioUrlPart=$1           os.getenv("SERVER_ADDRESS")
+# -DosioUsername=$2          os.getenv("OSIO_USERNAME")
+# -DosioPassword=$3          os.getenv("OSIO_PASSWORD")
+# -DcheWorkspaceName=$4      workspaceName
+# -DkeycloakToken=$5         theToken
+
+set +x
+
+# Script to execute: che-functional-tests/cico/run_EE_suite.sh
+git clone git@github.com:redhat-developer/che-functional-tests.git
```
Comment (pmacik) [inline booster_bdd/che.sh]: Please, use shallow clone to save load, bandwidth and disk space: `git clone --depth 1 git@github.com:redhat-developer/che-functional-tests.git`

File: booster_bdd/config/prod-preview.sh [hunk 2]
```diff
@@ -18,6 +18,9 @@ export WIT_API="${WIT_API:-https://api.prod-preview.openshift.io}"
 ## URI of the Openshift.io's Auth server
 export AUTH_API="${AUTH_API:-https://auth.prod-preview.openshift.io}"
 
+## URI of the Openshift.io's Che server
+export CHE_API="${CHE_API:-https://che.openshift.io}"
+
```
Comment (pmacik) [inline booster_bdd/config/prod-preview.sh]: The following variable used in `config/config.sh` is missing here:
```
## github repo URL (for use in creating Che workspace)
export GIT_REPO_URL="${GIT_REPO:-https://github.com/che-samples/console-java-simple.git}"

```

File: booster_bdd/config/config.sh [hunk 3]
```diff
@@ -89,6 +89,9 @@ export PIPELINE="${PIPELINE:-maven-releasestageapproveandpromote}"
 ## github repo name
 export GIT_REPO="${GIT_REPO:-test123}"
 
+## github repo URL (for use in creating Che workspace)
+export GIT_REPO_URL="${GIT_REPO:-https://github.com/che-samples/console-java-simple.git}"
+
```
Comment (pmacik) [inline booster_bdd/config/config.sh]: The following variable used in `config/prod-preview.sh` is missing here:
```
## URI of the Openshift.io's Che server
export CHE_API="${CHE_API:-https://che.openshift.io}"

```

Comment (ldimaggi) [reply booster_bdd/config/config.sh]: Changes made - many thanks!

File: booster_bdd/config/prod-preview.sh (hunk 2 above)
Comment (ldimaggi) [reply booster_bdd/config/prod-preview.sh]: Change made - thanks!

**Pull request merged**